example.json

```diff
@@ -15,6 +15,7 @@
     "mobileInstruction": "This is optional instruction text that will be shown when viewed on mobile.",
     "_setCompletionOn":"allItems",
     "_useGraphicsAsPins": false,
+    "_useNumbersAsPins": false,
```

Comment:
Thinking forward, having separate JSON attributes for the different styles doesn't feel like the most elegant implementation and can lead to mistakes if both attributes are enabled at the same time.

Would we be able to set up a singular JSON attribute, '_pinStyle', with different options, similar to '_setCompletionOn' that had a default, 'pins', but could be overridden by changing the value of the attribute to either 'tiles' or 'numbers'. This may require a bit of defensive programming to retain backwards compatibility but I believe it would help to clarify what these attributes do.

I'm keen to hear thoughts on this approach.